Remove nested directories from log rotation template

ryancragun opened the pull request:
nginx, redis_lb, and opscode-solr4 are the current services that use the
logrotate.erb template for their corresponding logrotate entries.  None
of those services actually have nested log directories which results in
an error that prevent logrotate from running correctly:

error: stat of /var/log/opscode/nginx/*/*.log failed: No such file or
directory

Removing unnecessary nested directory directive resolves this problem

stevendanna commented:
@ryancragun :+1: This looks good to me.  

Since this fixes a bug would you mind adding a short:

`ChangeLog-Entry: DESCRIPTION`

line to your commit message?

Out of curiostiy, is there a particular problem you were seeing this fail on?  I ask because that line subdirectory glob has been in place since the start of Chef Server 11.

ryancragun commented:
@stevendanna I added an entry to the CHANGELOG.md and updated the commit message.  As per the problem, when the incorrect glob was there the file would get rotated (eg: moved to access.log.0) but nginx would keep an open file handle on old log and continue writing to it even though it was renamed.  Strangely not even a USR1 to nginx would fix it.

stevendanna commented:
:+1: MWR

sdelano commented:
@ryancragun - Please rebase next time before merging. :heart:

ryancragun commented:
awh crap
